doc/netplan.md

```diff
@@ -808,6 +808,24 @@ Example:
 
      **Requires feature: sriov**
 
+``eswitch-mode`` (scalar) – since **0.104**
+
```

Comment:
'eswitch' seems jargony.  Would `embedded-switch-mode` be better?

Reply:
Thanks for your comments! Yes, I had thought about that variation as well and will change it accordingly.